src/debug/di/cordb.cpp

```diff
@@ -499,7 +506,7 @@ DbiGetThreadContext(HANDLE hThread,
     DT_CONTEXT *lpContext)
 {
     // if we aren't local debugging this isn't going to work
-#if !defined(_ARM_) || defined(FEATURE_DBGIPC_TRANSPORT_DI) || defined(__ANDROID__)
+#if !defined(_ARM_) || defined(FEATURE_DBGIPC_TRANSPORT_DI) || !HAVE_ALIGNED_MALLOC
```

Comment:
I'm not sure, but it seems that else branch in these functions is for remote debugging. Am I right? If so, how about using ``SUPPORT_REMOTE_DEBUGGING`` instead of ``HAVE_ALIGNED_MALLOC``?